Administrator Guide/arbiter-volumes-and-quorum.md

```diff
@@ -0,0 +1,219 @@
+Arbiter volumes and quorum options in gluster
+=============================================
+The arbiter volume is special subset of replica 3 volumes that is aimed at
+preventing split-brains and providing the same consistency guarantees as a normal
+replica 3 volumes without consuming 3x space. Before we look at how to create one
+and how they work etc. it is worthwhile to elaborate on the types of split-brain
+in gluster parlance and how features like server-quorum and client-quorum help in
+reducing the occurrence of split-brains to a certain extent.
+
+Split-brains in replica volumes
+================================
+When a file is in split-brain, there is an inconsistency in either data or
+metadata (permissions, uid/gid, extended attributes etc.)of the file amongst the
+bricks of a replica *and* we do not have enough information to authoritatively
+pick a copy as being pristine and heal to the bad copies, despite all bricks
+being up and online. For directories, there
+is also an entry-split brain where a file inside it has different gfids/
+file-type (say one is a file and another is a directory of the same name)
+across the bricks of a replica.
+
+https://github.com/gluster/glusterfs-specs/blob/master/done/Features/heal-info-and-split-brain-resolution.md
```

Comment:
It's better if this is a markdown link to a word. Something like:

[This document](https://github.com/gluster/glusterfs-specs/blob/master/done/Features/heal-info-and-split-brain-resolution.md) describes how to...